query.go

```diff
@@ -97,10 +97,10 @@ func getErrorFrameFromQuery(query *Query) data.Frames {
 	return frames
 }
 
-// query sends the query to the connection and converts the rows to a dataframe.
-func query(ctx context.Context, db Connection, converters []sqlutil.Converter, fillMode *data.FillMissing, query *Query) (data.Frames, error) {
+// QueryDB sends the query to the connection and converts the rows to a dataframe.
+func QueryDB(ctx context.Context, db Connection, converters []sqlutil.Converter, fillMode *data.FillMissing, query *Query, args ...interface{}) (data.Frames, error) {
```

Comment:
why not just `Query`?

Reply:
Unfortunately, [Query](https://github.com/grafana/sqlds/blob/0f7924cebd3edf6e66d353bce0b716d601fed907/query.go#L31) is already the name of the query unmarshaling struct.